pupil-spa/gen_config.sh

```diff
@@ -10,6 +11,12 @@ feedbackAPIErrorMaxAttempts=${CHECK_SUBMISSION_MAX_ATTEMPTS:-"3"}
 pupilPrefsAPIErrorDelay=${CHECK_SUBMISSION_ERROR_DELAY:-"3000"}
 pupilPrefsAPIErrorMaxAttempts=${CHECK_SUBMISSION_MAX_ATTEMPTS:-"3"}
 submissionPendingViewMinDisplay=${SUBMISSION_PENDING_MIN_DISPLAY:-"6000"}
+connectivityCheckViewMinDisplay=${CONNECTIVITY_CHECK_MIN_DISPLAY:-"6000"}
+testPupilConnectionQueueName=${TEST_PUPIL_CONNECTION_QUEUE_NAME:-"test-pupil-connection"}
+testPupilConnectionQueueUrl=${TEST_PUPIL_CONNECTION_QUEUE_URL:-"testPupilConnectionQueueUrlEnvValue"}
```

Comment:
i don' think the postfix of `envValue` is particularly appropriate / necessary on these 2 settings.

Reply:
Is "testPupilConnectionQueueUrlEnvValue" meant to be a deliberately invalid setting? If so, I would prefer something like 'example.com'.

Reply:
@GuyHarwood @jon-shipley the default values that gulp task will replace have been updated without the env part in the name.